source/_integrations/apple_tv.markdown

```diff
@@ -10,33 +10,126 @@ ha_release: 0.49
+There is currently support for the following entities within the Apple TV device:
 
-- Media Player
+- [Media Player](#media_player)
 - [Remote](#remote)
 
-## Configuration
+## Media Player
+
+The Apple TV media player platform will create a Media Player entity for each
+Apple TV discovered on your network.
+This entity will display the active app and playback controls.
+
+## Remote
+
+The Apple TV remote platform will automatically create a Remote entity for each Apple TV
+configured on to your Home Assistant instance.
+These entities allow you to turn the device on/off and to send control commands.
 
-Menu: *Configuration* > *Integrations*
+The following commands are currently available:
+
+- `wakeup`
+- `home`
+- `home_hold`
+- `top_menu`
+- `menu`
+- `select`
+- `up`
+- `down`
+- `left`
+- `right`
+- `volume_up`
+- `volume_down`
 
-Press on **Apple TV** and configure the integration:
+**NOTE:** Not all commands are supported by all Apple TV versions
 
-* Enter either an IP address or a device name and follow the next few steps
+### Service `send_command`
+
+| Service data<br>attribute | Optional | Description  |
+| ------------------------- | -------- | ------------ |
+| `entity_id`               | no       | `entity_id` of the Apple TV |
+| `command`                 | no       | Command, or list of commands to be sent |
+| `num_repeats`             | yes      | Number of times to repeat the commands |
+| `delay_secs`              | yes      | Interval in seconds between one send and another <br> This is a `float` value e.g. 1, 1.2 etc. |
+
+**Examples**
+
+Create a script to invoke the Netflix application based on the application icon
+being in a fixed place on the home screen:
+
+```yaml
+lounge_appletv_netflix:
+  alias: Select Netflix
+  sequence:
+  - service: remote.send_command
+    data:
+      entity_id: remote.lounge_appletv
+      delay_secs: 1.5
+      command:
+      - top_menu
+      - home
+      - right
+      - select
+```
+
+Script using the `home_hold` command to send your Apple TV to sleep and turn off
+the Media Player:
+
+```yaml
+apple_tv_sleep:
+  alias: Make the Apple TV sleep
+  sequence:
+    - service: remote.send_command
+      data:
+        entity_id: remote.lounge_appletv
+        delay_secs: 1
+        command:
+          - home_hold
+          - select
+    - service: media_player.turn_off
+      data:
+        entity_id:
+          - media_player.lounge_appletv
+```
+
+Send 3 `left` commands with delay between each:
+
+```yaml
+service: remote.send_command
+data:
+  entity_id: remote.apple_tv
+  num_repeats: 3
+  delay_secs: 2.5
+  command:
+    - left
+```
+
+## Configuration
+
+First add the Apple TV integration to your Home Assistant setup by clicking on
+the "_+ ADD INTEGRATION_" button on the *Integrations* screen.
+Type "Apple TV" in the _Search integrations_ field and click on the *Apple TV*
+device.
+Now follow the instructions present on screen.
```

Comment:
Let's move this up. It should be the first chapter after the introduction text.

Reply:
This item was resolved without a change or comment, did I miss something?

Reply:
Sorry. My bad. I'll move it and add a commit.